.github/workflows/build-ws-container.yml

```diff
@@ -0,0 +1,36 @@
+name: build-ws-container
+on:
+  # can be run manually on https://github.com/cockpit-project/cockpit/actions
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    environment: quay.io
+    permissions: {}
+    timeout-minutes: 20
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Log into container registry
+        run: docker login -u ${{ secrets.QUAY_BOTUSER }} -p ${{ secrets.QUAY_TOKEN }} quay.io
+
+      - name: Build and push ws container
+        run: |
+          TAG=$(date +%Y-%m-%d)
+          docker build -t quay.io/cockpit/ws:$TAG containers/ws
```

Comment:
I intentionally used docker here. In the past we had some problems with older docker versions not being able to read podman-created containers.